### PR TITLE
ELEMENTS-1499: wait for dropdownButton resize

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: ui/test/nuxeo-actions-menu.test.js [hunk 1]
```diff
@@ -122,6 +122,10 @@ suite('nuxeo-actions-menu', () => {
     await flush();
   }
 
+  async function waitMenuRedered(menu) {
```
Comment (semisse) [inline ui/test/nuxeo-actions-menu.test.js]: nit: 
```suggestion
  async function waitMenuRendered(menu) {
```

Comment (Gabez0r) [reply ui/test/nuxeo-actions-menu.test.js]: Thanks. Fixed. 👍

File: ui/widgets/nuxeo-actions-menu.js [hunk 2]
```diff
@@ -205,47 +206,57 @@ import './nuxeo-tooltip.js';
...
       });
+      enqueueDebouncer(this.__layoutDebouncer);
+    }
+
+    async _distributeNodes() {
+      if (this._counter === undefined) this._counter = 0;
+      let els = this._getDropdownElements();
+      /**
+       * XXX: We're using this.contentWidth instead of this.scrollWidth because it takes too much time to be
+       * updated on polyfilled browsers (Firex and Edge), leading to an empty menu if there's a single element
```
Comment (semisse) [inline ui/widgets/nuxeo-actions-menu.js]:
```suggestion
       * updated on polyfilled browsers (Firefox and Edge), leading to an empty menu if there's a single element
```

Comment (Gabez0r) [reply ui/widgets/nuxeo-actions-menu.js]: Fixed! Thank you. 👍

File: ui/test/nuxeo-actions-menu.test.js [hunk 3]
```diff
@@ -76,10 +86,6 @@ suite('nuxeo-actions-menu', () => {
     );
   }
 
-  function actionsMenu(el) {
```
Comment (Gabez0r) [inline ui/test/nuxeo-actions-menu.test.js:79]: Just moved this up.

**Review by richardsd — COMMENTED** [review]
Lgtm, but wasn't this causing some visual artifacts?

**Review by semisse — APPROVED** [review]

**Comment by Gabez0r** [comment]
> Lgtm, but wasn't this causing some visual artifacts?

@richardsd I didn't notice any. @semisse did you?

**Review by richardsd — APPROVED** [review]

**Comment by semisse** [comment]
> > Lgtm, but wasn't this causing some visual artifacts?
> 
> @richardsd I didn't notice any. @semisse did you?

I tested the UI in different scenarios and I could not see any side effects with this change.

**Pull request merged**